examples/server_profile.rb

```diff
@@ -9,16 +9,20 @@
 # CONDITIONS OF ANY KIND, either express or implied. See the License for the
 # specific language governing permissions and limitations under the License.
 
+# NOTES:
+#  - It is needed configure previously some things like the network connections, storage pool, storage system, etc., to run this example.
```

Comment:
Probably better to be specific here:

This example requires the following resources to be available in the appliance:
- FC Networks: 'FC1', 'FC2'
- Storage Pool: (...)

Reply:
@fgbulsoni Maybe the below text is good, it is simple to understand?
```ruby
# NOTES:
# This example requires the following resources to be available in the appliance:
#  - FC Network: 'FCNetwork1'
#  - FCoE Network: 'fcoe1'
#  - Ethernet Network: 'EthernetNetwork1'
#  - Server Profile Template: 'spt1'
#  - Server Hardware Type: 'BL660c Gen9 1'
#  - Server Hardware: 'Encl1, bay 2'
#  - Enclosure Group: 'EG_1'
# To create volume attachments:
#  - The attributes file "volume_attachments_variables" is loading variables to be used in this example.
#  - The Storage System must have at least one connection using 'FCNetwork1' and this same network network must have uplinkSet connected on the Interconnect,
#   and the Server Hardware related to that network network is the Server Hardware used in this example.
```

Reply:
(...) this same *network network* (...)
must have **an** uplinkSet (...)
(...)Server Hardware related to that *network network*
Also I'd suggest using names following a pattern. i.e. stick to either 
- `FCNetwork1`, `FCoENetwork1` 
or 
- `fc1`, `fcoe1`
, etc. Not both styles.